hammer_cli_foreman_rh_cloud.gemspec

```diff
@@ -13,9 +13,11 @@ Gem::Specification.new do |spec|
   spec.platform      = Gem::Platform::RUBY
   spec.summary       = 'Foreman Rh Cloud plugin for Hammer CLI'
 
-  spec.files         = Dir['{lib,config}/**/*', 'LICENSE', 'README*'] + Dir["locale/**/*.{po,pot,mo}"]
+  spec.files         = Dir['{lib,config}/**/*', 'LICENSE', 'README*'] + Dir['locale/**/*.{po,pot,mo}']
   spec.require_paths = ['lib']
   spec.test_files    = Dir['{test}/**/*']
 
   spec.add_dependency 'hammer_cli_foreman', '>= 3.0.0', '< 4.0.0'
+
+  s.required_ruby_version = '>= 2.7', '< 4'
```

Comment:
This broke because it needs to be `spec`: https://github.com/theforeman/hammer-cli-foreman-rh-cloud/pull/7

Reply:
Thanks for fixing it @ekohl :)